frontend/src/AppState/AppState.tsx

```diff
@@ -28,6 +28,18 @@ type AppStateContextState = {
     appState: AppState;
 };
 
+// Define the type
+class GitHash {
```

Comment:
Could we rename this class to fingerprint to maintain consistency, readability and support?